ui/templates/connectionTemplates.json

```diff
@@ -0,0 +1,208 @@
+{
+  "AMQP 0.9.1": {
+    "name": "AMQP 0.9.1",
+    "connectionType": "amqp-091",
+    "connectionStatus": "closed",
+    "failoverEnabled": true,
+    "uri": "amqp://user:password@localhost:5672/vhost",
+    "sources": [
+      {
+        "addresses": [
+          "queueName"
+        ],
+        "authorizationContext": [
+          "ditto:inbound-auth-subject",
+          "..."
+        ]
+      }
+    ],
+    "targets": [
+      {
+        "address": "exchangeName/routingKey",
+        "topics": [
+          "_/_/things/twin/events",
+          "_/_/things/live/messages"
+        ],
+        "authorizationContext": [
+          "ditto:outbound-auth-subject",
+          "..."
+        ]
+      }
+    ]
+  },
+  "AMQP 1.0": {
+    "name": "AMQP 1.0",
+    "connectionType": "amqp-10",
+    "connectionStatus": "closed",
+    "failoverEnabled": true,
+    "uri": "amqps://user:password@hono.eclipse.org:5671",
+    "sources": [
+      {
+        "addresses": [
+          "telemetry/FOO"
+        ],
+        "authorizationContext": [
+          "ditto:inbound-auth-subject"
+        ]
+      }
+    ],
+    "targets": [
+      {
+        "address": "events/twin",
+        "topics": [
+          "_/_/things/twin/events"
+        ],
+        "authorizationContext": [
+          "ditto:outbound-auth-subject"
+        ]
+      }
+    ]
+  },
+  "MQTT 3.1.1": {
+    "name": "MQTT 3.1.1",
+    "connectionType": "mqtt",
+    "connectionStatus": "closed",
+    "failoverEnabled": true,
+    "uri": "tcp://test.mosquitto.org:1883",
+    "sources": [
+      {
+        "addresses": [
+          "eclipse-ditto-sandbox/#"
+        ],
+        "authorizationContext": [
+          "ditto:inbound-auth-subject"
+        ],
+        "qos": 0,
+        "filters": []
+      }
+    ],
+    "targets": [
+      {
+        "address": "eclipse-ditto-sandbox/{{ thing:id }}",
+        "topics": [
+          "_/_/things/twin/events"
+        ],
+        "authorizationContext": [
+          "ditto:outbound-auth-subject"
+        ],
+        "qos": 0
+      }
+    ]
+  },
+  "MQTT 5": {
+    "name": "MQTT 5",
+    "connectionType": "mqtt-5",
+    "connectionStatus": "closed",
+    "failoverEnabled": true,
+    "uri": "tcp://test.mosquitto.org:1883",
+    "sources": [
+      {
+        "addresses": [
+          "eclipse-ditto-sandbox/#"
+        ],
+        "authorizationContext": [
+          "ditto:inbound-auth-subject"
+        ],
+        "qos": 0,
+        "filters": []
+      }
+    ],
+    "targets": [
+      {
+        "address": "eclipse-ditto-sandbox/{{ thing:id }}",
+        "topics": [
+          "_/_/things/twin/events"
+        ],
+        "authorizationContext": [
+          "ditto:outbound-auth-subject"
+        ],
+        "qos": 0
+      }
+    ]
+  },
+  "HTTP 1.1": {
+    "name": "HTTP 1.1",
+    "connectionType": "http-push",
+    "connectionStatus": "closed",
+    "failoverEnabled": true,
+    "uri": "http://user:password@localhost:80",
```

Comment:
Great idea. I would prefer to put some placeholder instead of everybody sharing the same uuid?
`<YOUR-WEBHOOK-ID>`
You prefer to have something that works out of the box without editing, right?

Reply:
You're right .. we should use a placeholder instead ;)